### ISSUE TITLE
drive not available in AUR (Arch User Repository)

### ISSUE THREAD

**Issue by avila** [odeke-em/drive#1042]
drive seems to have gone missing from AUR:

`yaourt -S drive`
`error: target not found: drive`

the [Platform Packages](https://github.com/odeke-em/drive/blob/master/platform_packages.md) information is, therefore, outdated.

I would appreciate an indication of other means of installing under Arch-based distros.

**Comment by cgxeiji** [comment]
You can install `drive` using Go.

Follow the [Golang installation guide](https://golang.org/doc/install), make sure you have the path configured:

```
cat << ! >> ~/.bashrc
> export GOPATH=\$HOME/gopath
> export PATH=\$GOPATH:\$GOPATH/bin:\$PATH
> !
source ~/.bashrc # To reload the settings and get the newly set ones # Or open a fresh terminal
```

and do:

```
go get -u github.com/odeke-em/drive/cmd/drive
```

Source: [drive installation guide](https://github.com/odeke-em/drive#installing), I'm currently using drive in Arch.

**Comment by cniw** [comment]
@mr-avila you can use instead `yaourt -S drive-bin`
i found it from this thread https://bbs.archlinux.org/viewtopic.php?id=240413

**Comment by m-svo** [comment]
@mr-avila this should be fixed now with b76b1f2f1ae8ee35b1ce76fa28b230718b8b5db4

**Comment by odeke-em** [comment]
Great, thanks everyone, thank you @avila for the report, @cgxeiji @cniw @m-svo for chiming in! The issue was reported as fixed by @m-svo in https://github.com/odeke-em/drive/issues/1042#issuecomment-503914082 and so I shall close this one.
